src/services/dumpers/templates/agent-nodejs/docker-compose.hbs

```diff
@@ -0,0 +1,34 @@
+version: '3.4'
+services:
+  app:
+    build:
+      context: .
+      dockerfile: Dockerfile
+{{#if network}}
+      network: {{network}}
+{{/if}}
+    container_name: {{ containerName }}
+    environment:
```

Comment:
Why not using the `.env`/`env_file` file to prevent potential issues ?

https://docs.docker.com/compose/environment-variables/set-environment-variables/#use-the-env_file-attribute. If feels like we are only re-binding value from .env anyway

Reply:
👍